pkg/volume/glusterfs/glusterfs.go

```diff
@@ -1056,3 +1063,18 @@ func parseClassParameters(params map[string]string, kubeClient clientset.Interfa
 	}
 	return &cfg, nil
 }
+
+type glusterfsDiskExpander struct {
+	*glusterfs
+}
+
+var _ volume.Expander = &glusterfsDiskExpander{}
+
+func (e *glusterfsDiskExpander) ExpandVolumeDevice(newSize resource.Quantity, oldSize resource.Quantity) error {
+	glog.Infof("********** Calling expand volume device **********")
```

Comment:
@humblec do you think you will have time to implement the change needed here for the glusterfs volume type? I have already verified that, this function gets called when a glusterfs volume is resized. We just need to implement correct behaviour here. let me know.

Reply:
@gnufied Sure.. !! I am on it.

Reply:
https://github.com/humblec/kubernetes/commit/ecde2183594ec520bc03f7ded04bcd6c62f36766 Done :) Thanks!